libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java

```diff
@@ -173,21 +173,26 @@ public Bundle getTranslations() {
             }
 
             String fieldName = stringField.getName();
-            // Filter out all strings that are not prefixed with `gutenberg_mobile_`
-            if (!fieldName.startsWith("gutenberg_mobile_")) {
+            // Filter out all strings that are not prefixed with `gutenberg_native_`
+            if (!fieldName.startsWith("gutenberg_native_")) {
                 continue;
             }
 
-            // Add the mapping english => [ translated ] to the bundle if both string are not empty
-            String currentResourceString = currentResources.getString(resourceId);
-            String defaultResourceString = defaultResources.getString(resourceId);
-            if (currentResourceString.length() > 0 && defaultResourceString.length() > 0) {
-                translations.putStringArrayList(
-                        defaultResourceString,
-                        new ArrayList<>(Arrays.asList(currentResourceString))
-                );
+            try {
+                // Add the mapping english => [ translated ] to the bundle if both string are not empty
+                String currentResourceString = currentResources.getString(resourceId);
+                String englishResourceString = englishResources.getString(resourceId);
+                if (currentResourceString.length() > 0 && englishResourceString.length() > 0) {
+                    translations.putStringArrayList(
+                            englishResourceString,
+                            new ArrayList<>(Arrays.asList(currentResourceString))
+                    );
+                }
+            } catch (Resources.NotFoundException rnfe) {
+                AppLog.e(T.EDITOR, rnfe);
```

Comment:
Based on https://github.com/wordpress-mobile/WordPress-Android/pull/10810/#issuecomment-556008550, it seems that getting an exception is almost expected in some cases. What do you think about demoting the log level to warning instead of error? That way, the log entry (also accessible from inside the app and on Support tickets) will not be as alarming.

Reply:
Good point 👍 maybe event `information` level?

Reply:
Went with warning 👍